src/_pytest/assertion/rewrite.py

```diff
@@ -412,6 +412,10 @@ def _rewrite_test(config, fn):
         return None, None
     rewrite_asserts(tree, fn, config)
     try:
+        import astunparse
+
+        with open(fn + "UNPARSED", "w", encoding="utf-8") as file:
```

Comment:
is the unconditional unparse there for debugging?

Reply:
Yes it was but already removed